quantum/quantum.c

```diff
@@ -481,3 +481,39 @@ void api_send_unicode(uint32_t unicode) {
 __attribute__((weak)) void startup_user() {}
 
 __attribute__((weak)) void shutdown_user() {}
+
+// Useful string manipulation code
+const char *get_u8_str(uint8_t curr_num, char curr_pad) {
+    static char    buf[4]   = {0};
+    static uint8_t last_num = 0xFF;
+    static char    last_pad = '\0';
```

Comment:
```suggestion
    static char    buf[4]   = {0};
    static uint8_t last_num = 0;
    static char    last_pad = '\0';
```
The only thing which makes this safe is the fact that `last_pad = '\0'` is an invalid value, so you may as well initialize `last_num` to 0 (then it can go into BSS instead of initialized data, saving a byte of flash).